### PR TITLE
Fix fetching of accounts

### PR DESCRIPTION
**Pull request by tomusdrw**
Asking for accounts synchronously was not always working.

Additionally sets defaultAccount when using global `web3` object.

### CODE REVIEW

File: web3/content_script.js [hunk 1]
```diff
@@ -105,6 +105,7 @@ function main () {
         origin
       }, (result) => {
         if (!result) {
+          retry(ev);
```
Comment (ngotchac) [inline web3/content_script.js:118]: Why would there be no result ? If it's a network or overload issue, we might want to add a small timeout ?

**Review by ngotchac — COMMENTED** [review]
Merge issue and a question, but otherwise looks good

File: web3/content_script.js (hunk 1 above)
Comment (tomusdrw) [reply web3/content_script.js:118]: > The JSON response object sent by the handler of the message. If an error occurs while connecting to the extension, the callback will be called with no arguments and runtime.lastError will be set to the error message.

I've seen no results here right after refreshing the page, not sure what's causing this. Will check `runtime.lastError` for some details, but perhaps the background script is not ready yet?

Comment (ngotchac) [reply web3/content_script.js:118]: Okay yes, that makes sense !

**Review by ngotchac — APPROVED** [review]
Merged master. Looks good to me !

**Pull request merged**